codalab/lib/worksheet_util.py

```diff
@@ -938,8 +939,10 @@ def get_command(value_obj):  # For directives only
         finally:
             last_was_empty_line = new_last_was_empty_line
 
+    if len(raw_to_interpreted) != len(raw_items):
```

Comment:
Put a TODO here, because this really shouldn't happen...we can fix it after the demo.